package.json

```diff
@@ -42,6 +42,7 @@
     "preact": "^10.5.14",
     "react": "17.0.2",
     "react-dom": "17.0.2",
+    "react-transition-group": "^4.4.2",
```

Comment:
```suggestion
    "react-transition-group": "^4.4.2",
```

Use `Transition` in [Headless UI](https://headlessui.dev/react/transition) instead.